imsvdex/vdex.py

```diff
@@ -84,7 +83,7 @@ def __init__(
           should the term be returned in the default language
           or as self.unnamed_term ?
         """
-        if isinstance(matrix, basestring):
+        if isinstance(matrix, str):
```

Comment:
This is not correct because it is not matching unicode on Python2 anymore.
`basestring` should become `six.string_types`.